nsxt/resource_nsxt_policy_group_test.go

```diff
@@ -60,6 +61,50 @@ func TestAccResourceNsxtPolicyGroup_singleIPAddressCriteria(t *testing.T) {
 	})
 }
 
+func TestAccResourceNsxtGlobalPolicyGroup_singleIPAddressCriteria(t *testing.T) {
+	name := fmt.Sprintf("test-nsx-global-policy-group-ipaddrs")
+	updatedName := fmt.Sprintf("%s-update", name)
+	testResourceName := "nsxt_policy_group.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccNsxtGlobalPolicyGroupPrecheck(t) },
```

Comment:
you can add a list of precheck functions here, instead of defining a special one (there is already one for global manager and one for site)

Reply:
Fixed. Thanks!